modules/osd-create-cluster-red-hat-account.adoc

```diff
@@ -0,0 +1,125 @@
+= Creating a cluster on GCP with a Red Hat cloud account
+endif::osd-on-gcp[]
+
+Through {OCM}, you can create an {product-title} cluster 
+ifdef::osd-on-aws[]
+on {AWS} 
+endif::osd-on-aws[]
+ifdef::osd-on-gcp[]
+on {GCP} 
+endif::osd-on-gcp[]
+using a standard cloud provider account owned by Red Hat.
+
+.Procedure
+
+. Log in to {console-redhat-com} and click *Create Cluster*.
+
+. In the *Cloud* tab, click *Create Cluster* in the *Red Hat OpenShift Dedicated* row.
+
+. In the *Billing model* dialog, configure the subscription type and infrastructure type:
+.. Select a subscription type.
+.. Select the *Red Hat cloud account* infrastructure type to deploy {product-title} in a cloud provider account that is owned by Red Hat.
+.. Click *Next*.
+
+ifdef::osd-quickstart[]
+. Select *Run on Amazon Web Services* or *Run on Google Cloud Platform* to choose a cloud infrastructure provider 
+endif::osd-quickstart[]
+ifdef::osd-on-aws[]
+. Select *Run on Amazon Web Services* 
+endif::osd-on-aws[]
+ifdef::osd-on-gcp[]
+. Select *Run on Google Cloud Platform* 
+endif::osd-on-gcp[]
+and click *Next*.
+
+. On the *Cluster details* page, provide a name for your cluster and specify the cluster details:
+.. Add a *Cluster name*.
+.. Select a cluster version from the *Version* drop-down menu.
+.. Select a cloud provider region from the *Region* drop-down menu.
+.. Select a *Single zone* or *Multi-zone* configuration.
+.. Select a *Persistent storage* capacity and state the number of *Load balancers* that you require for your cluster.
+.. Leave *Enable user workload monitoring* selected to monitor your own projects in isolation from Red Hat Site Reliability Engineer (SRE) platform metrics. The option is enabled by default.
+.. Optional: Select *Enable additional etcd encryption* if you require etcd key value encryption. With this option, the etcd key values are encrypted, but not the keys. The option is in addition to the control plane storage encryption that encrypts the etcd volumes in {product-title} clusters by default.
++
+[NOTE]
+====
+By enabling etcd encryption for the key values in etcd, you will incur a performance overhead of approximately 20%. The overhead is a result of introducing this second layer of encryption, in addition to the default control plane storage encryption that encrypts the etcd volumes. Consider enabling etcd encryption only if you specifically require it for your use case.
+====
+.. Click *Next*.
+
+. On the *Default machine pool* page, select a *Compute node instance type* and a *Compute node count*. The number and types of nodes that are available depend on your {product-title} subscription. If you are using multiple availability zones, the compute node count is per zone.
++
+[NOTE]
+====
+After your cluster is created, you can change the number of compute nodes in your cluster, but you cannot change the compute node instance type. The number and types of nodes available to you depend on your {product-title} subscription.
```

Comment:
This is saying that a cluster can only have 1 node type.  I am confirming this, but I know for CCS and ROSA you can add a new machine pool with different node types. Meaning that the node types have to be the same in the pool, but can have different types in different pools in the cluster